openmdao/drivers/tests/test_genetic_algorithm_driver.py

```diff
@@ -1381,9 +1381,8 @@ def compute(self, inputs, outputs):
         prob = om.Problem()
         prob.model.add_subsystem('cylinder', Cylinder(), promotes=['*'])
 
-        indeps = prob.model.add_subsystem('indeps', om.IndepVarComp(), promotes=['*'])
-        indeps.add_output('radius', 2.)  # height
-        indeps.add_output('height', 3.)  # radius
+        prob.model.set_input_defaults('radius', 2.)
```

Comment:
I believe you call `set_val` on this too?

Reply:
fixed